cmd/crc/cmd/start.go

```diff
@@ -243,6 +244,12 @@ Use the 'oc' command line interface:
   {{ .CommandLinePrefix }} {{ .EvalCommandLine }}
   {{ .CommandLinePrefix }} oc login -u {{ .ClusterConfig.DeveloperCredentials.Username }} {{ .ClusterConfig.URL }}
 `
+	startTemplateForPodman = `Enabled the podman socket.
```

Comment:
I don't think this is a useful message for the user. What does this mean? What matters is that there's a podman runtime (dunno about the correct name) that is available and ready to interact with.